lib/bolt/pal/yaml_plan/step/script.rb

```diff
@@ -6,35 +6,34 @@ class YamlPlan
       class Step
         class Script < Step
           def self.allowed_keys
-            super + Set['script', 'parameters', 'arguments']
+            super + Set['arguments']
           end
 
-          def self.required_keys
-            Set['targets']
+          def self.option_keys
+            Set['catch_errors', 'env_vars', 'run_as']
```

Comment:
This is an issue for Puppet plans too, but we don't validate that `env_vars` is a hash before trying to operate on it. Totally up to you if you want to ticket it, or solve it here.
Plan:
```
  - name: script_step
    script: /home/lucy/githubs/bolt/tasks/init.sh
    env_vars: ''
    targets: $targets
```
Output:
```
...
Failed on localhost:20024:
  undefined method `map' for "":String
  Did you mean?  tap
Failed on localhost:20025:
  undefined method `map' for "":String
  Did you mean?  tap
Failed on 2 targets: localhost:20024,localhost:20025
...
```

Reply:
I'll ticket this separately.